scripts/apps/authoring/translations/translationsWidget.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import {RelativeDate} from 'core/datetime/relativeDate';
+import {state as State} from 'apps/search/components/fields/state';
+import {connectServices} from 'core/helpers/ReactRenderAsync';
+import {IArticle} from 'superdesk-interfaces/Article';
+
+interface IProps {
+    item: IArticle;
+    $filter: any;
+    gettextCatalog: any;
+    datetime: any;
+    api: any;
+    authoringWorkspace: any;
+    TranslationService: any;
+}
+
+interface IState {
+    translations: Array<IArticle>;
+    references: Dictionary<IArticle['_id'], IArticle>;
+}
+
+function getReferences(api, translations: Array<IArticle>) {
+    return new Promise<Array<IArticle>>((resolve) => {
+        const translated_from_ids = translations
+            .map((translation) => translation.translated_from)
+            .filter((translated_from_id) => translated_from_id != null); // is null for original
+
+        api('archive').query({
+            $or: translated_from_ids.map((id) => ({_id: id})),
+        }).then((response) => {
+            resolve(response._items);
+        });
+    });
+}
+
+class TranslationsWidgetComponent extends React.Component<IProps, IState> {
+    static propTypes: any;
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            translations: null,
+            references: null,
+        };
+    }
+
+    componentDidMount() {
+        const {api, item, TranslationService} = this.props;
+
+        TranslationService.getTranslations(item).then((response) => {
+                const translations: Array<IArticle> = response._items;
+
+                getReferences(api, translations).then((references) => {
+                    this.setState({
+                        translations: response._items,
+                        references: references.reduce((result, reference) => {
+                            result[reference._id] = reference;
+                            return result;
+                        }, {}),
+                    });
+                });
+            });
+    }
+
+    render() {
+        const {$filter, gettextCatalog, authoringWorkspace, datetime} = this.props;
+        const {translations, references} = this.state;
+
+        if (translations == null) {
+            return null;
+        }
+
+        const sortOriginalFirst = (a: IArticle) => a.translated_from == null ? -1 : 1;
```

Comment:
in theory you could sort by firstcreated that should make the order stable for other translations

Reply:
`firstcreated` is the same for all translations.

Reply:
that's weird, imo it should be different

Reply:
Created `SDESK-3487` for this.